docs/pages/10_view.adoc

```diff
@@ -0,0 +1,629 @@
+ifndef::env-site,env-github[]
+include::../_attributes.adoc[]
+endif::[]
+// Settings
+:idprefix:
+:idseparator: -
+:icons: font
+:KW: [purple]##**
+:KWE: **##
+
+[judo-req="true",judo-req-id="REQ-SRV-014"]
+= User Interface
+:page-toctitle:User Interface
+
+Views and rows play a crucial role in defining the visual representation of data. Views are special transfer objects that not only carry data but also define how this data is presented to the user. Rows, on the other hand, are a type of transfer object specifically designed for visualizing data in a tabular format.
```

Comment:
Current ESM universe has the following features which are missing in JSL:

- profile: (optional) points to the view of the principal
- dashboard: selects an access view which is the dashboard / landing page of the application

Reply:
https://blackbelt.atlassian.net/browse/JNG-5642
https://blackbelt.atlassian.net/browse/JNG-5643